app/app.js

```diff
@@ -23,14 +23,12 @@
     { name: 'Render Complex List',       path: '/render-complex-list' }
   ];
 
-  var HANDLEBARS_DEFAULT = "/ember/handlebars-v1.3.0.js";
-
   var EMBER_VERSIONS = [
-    { name: '1.9.1',                     path: '/ember/ember.prod-1.9.1.js',       handlebarsPath: '/ember/handlebars-v2.0.0.js'},
-    { name: '1.10.0-beta.1+canary',      path: '/ember/1.10.0-beta.1.canary.js',   handlebarsPath: '/ember/handlebars-v2.0.0.js'},
-    { name: '1.9.0-beta.3',              path: '/ember/1.9.0-beta.3.js',           handlebarsPath: '/ember/handlebars-v2.0.0.js'},
-    { name: '1.8.1',                     path: '/ember/1.8.1.js',                  handlebarsPath: HANDLEBARS_DEFAULT},
-    { name: '1.7.1',                     path: "/ember/1.7.1.js",                  handlebarsPath: HANDLEBARS_DEFAULT}
+    { name: '1.9.1',                     path: '/ember/ember.prod-1.9.1.js',       compilerPath: '/ember/handlebars-v2.0.0.js'},
+    {name: '1.10.0-beta.3',              path: "/ember/1.10.0-beta.3.js",          compilerPath: "/ember/no-htmlbars.js" },
```

Comment:
Also, if you make the `compilerPath` actually be `ember-template-compiler.js` (the one that is distributed via Bower and to S3 along with the build) then this is "future proof" for when I remove the compiler in beta.4 or beta.5.